launch/light_painting.launch

```diff
@@ -17,4 +17,9 @@
 
     <node name="$(anon rviz)" pkg="rviz" type="rviz" respawn="false" args = "-d $(find light_painting)/config/moveit.rviz" />
    
+    <!-- Launch Arduino Connection -->
+    <include file="$(find light_painting)/launch/t2_led_paintbrush.launch" >
+        <arg name="port" value="/dev/ttyACM0"/>
```

Comment:
Need to find a better way of automatically selecting the correct port.
This method requires manually setting the USB port address. Arduino typically switches back and forth between `/dev/ttyACM0` and `/dev/ttyACM1`

Reply:
Include in future feature. Not critical for this PR.